shipyard_blueprints/smartsheets/shipyard_smartsheets/cli/upload.py

```diff
@@ -0,0 +1,372 @@
+
+
+def upload_replace(
+    smart: smartsheet.Smartsheet,
+    logger: logging.Logger,
+    file_path: str,
+    name: str,
+    file_type: str,
+    sheet_id: Optional[str] = None,
+):
+    """This function uploads a file (read in as a pandas dataframe) to Smartsheet. If the sheet_id is provided, the existing sheet will be overwritten (if the datatypes) match, otherwise a new sheet will be created
+    Args:
+        smart: The smartsheet client
+        logger: The logger to STDOUT
+        file_path: The file path of target file to be uploaded
+        name: The name of the sheet to be created
+        file_type: The type of file being loaded (csv or xlsx)
+        sheet_id: The ID of the sheet to be overwritten
+
+    Returns: The response from the smartsheet API
+
+    """
+    try:
+        # NOTE: if sheet exists, then update the rows, otherwise create a new one
+        if not sheet_id:
+            if file_type == "csv":
+                response = smart.Sheets.import_csv_sheet(
+                    file=file_path,
+                    sheet_name=name,
+                    header_row_index=0,
+                    primary_column_index=0,
+                )
+            else:
+                response = smart.Sheets.import_xlsx_sheet(
+                    file=file_path, sheet_name=name
+                )
+        else:
+            data = read_data(file_path, file_type)
+            sheet = smart.Sheets.get_sheet(sheet_id)
+            column_mapping = map_columns(smart, sheet_id)
+            new_rows = form_rows(smart, column_mapping, data, insert_method="replace")
+            if file_type == "csv":
+                # clear the existing sheet content
+                delete_sheet_contents(smart, logger, sheet_id)
+                response = smart.Sheets.add_rows(sheet.id, new_rows)
+            elif file_type == "xlsx":
+                # get the sheet_data
+                response = smart.Sheets.add_rows(sheet.id, new_rows)
+    except FileNotFoundError:
+        raise (
+            ExitCodeException(
+                f"Error when trying to read in data, file {file_path} was not found",
+                exit_code=ss.EXIT_CODE_FILE_NOT_FOUND,
+            )
+        )
+
+    except Exception as e:
+        raise (
+            ExitCodeException(
+                f"Error in running replace job. {str(e)}",
+                exit_code=ss.EXIT_CODE_UPLOAD_ERROR,
+            )
+        )
+    else:
+        return response
+
+
+def main():
+    args = get_args()
+    logger = get_logger()
+    try:
+        if args.folder_name != "":
+            file_path = os.path.join(args.folder_name, args.file_name)
+        else:
+            file_path = args.file_name
+
+        # establish client and check cif access token is valid
+        smart = smartsheet.Smartsheet(args.access_token)
+        sheet_name = args.sheet_name if args.sheet_name != "" else None
+        sheet_id = None if args.sheet_id == "" else args.sheet_id
```

Comment:
small cosmetic change. Can you have the logic be handled the same way for the sheet_name and sheet_id
```
sheet_name = args.sheet_name if args.sheet_name != "" else None
sheet_id = args.sheet_id if args.sheet_id != "" else None
```
or 
```
sheet_name = None if args.sheet_name == "" else args.sheet_name
sheet_id = None if args.sheet_id == "" else args.sheet_id
```